base/applications/wusa/wusa.c

```diff
@@ -0,0 +1,14 @@
+#define WIN32_NO_STATUS
+#define WIN32_LEAN_AND_MEAN
+#include <windows.h>
+#include <tchar.h>
+
+int
+WINAPI
+_tWinMain(HINSTANCE hCurInst,
```

Comment:
You can make this wWinMain for simplicity. TCHARs aren't really very useful these days.